task-list/app/views/people/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1> All of <%= @person.name %>'s Uncompleted Tasks:</h1>
+
+<ul>
+  <% @tasks.each do |task| %>
+    <% if task.completed_date == nil %>
```

Comment:
We have a check like this throughout the codebase, and we could "Metzatize" it (very, very slightly) by moving it into the Task model class:
```ruby
class Task < ActiveRecord::Base
  def completed?
    return completed_date == nil
  end
end
```

That way our view logic doesn't have to "know" that a task is considered incomplete specifically when that field is nil. In this case its a very subtle difference, I think it's good practice for separating the concerns between Model and View in the MVC pattern. We'll start having more complex model logic soon and the distinction will probably become more clear over time.